c2corg_ui/static/menu-discourse/menu.html

```diff
@@ -0,0 +1,117 @@
+## HEAD
+<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">
+
+## En-tete
+<aside id="sidemenu">
+  <div class="page-menu">
+    <a href="//www.demov6.camptocamp.org">
```

Comment:
it does seem to work. The question is if we prefer to suddenly use this element tag (that we never used anywhere else) in a separate file, or to have `var baseUrl = '//www.demov6.camptocamp.org/';` in the same HTML file?